private/buf/bufcli/bufcli.go

```diff
@@ -547,10 +547,7 @@ func NewContextModifierProvider(
 		}
 		return func(ctx context.Context) context.Context {
 			ctx = bufrpc.WithOutgoingCLIVersionHeader(ctx, Version)
-			if token != "" {
-				return rpcauth.WithToken(ctx, token)
-			}
-			return ctx
+			return rpcauth.WithTokenIfNoneSet(ctx, token)
```

Comment:
I guess I could in-line the `bufrpc.WithOutgoingCLIVersionHeader` call here also, but I'm not sure that's necessary...